pandas/io/excel/_util.py

```diff
@@ -23,32 +23,46 @@ def register_writer(klass):
     _writers[engine_name] = klass
 
 
-def get_default_writer(ext):
+def get_default_engine(ext, mode="read"):
     """
-    Return the default writer for the given extension.
+    Return the default reader/writer for the given extension.
 
     Parameters
     ----------
     ext : str
         The excel file extension for which to get the default engine.
+    mode : str
+        Whether to get the default engine for reading or writing.
+        Either 'read' or 'write'
 
     Returns
     -------
     str
         The default engine for the extension.
     """
+    _default_readers = {
+        "xlsx": "openpyxl",
```

Comment:
Looks to me like this changes current behavior. On master, if xlrd is installed and version < 2.0 and openpyxl is not, will currently default to xlrd. In this PR, this will error instead, breaking user code.

Reply:
Fixed and tested it with xlrd==1.2.0 and no openpyxl.